socorro/external/es/new_crash_source.py

```diff
@@ -53,34 +54,30 @@ def new_crashes(self, date, product, versions):
         """
         next_day = date + datetime.timedelta(days=1)
 
-        query = {
-            'filter': {
-                'bool': {
-                    'must': [
-                        {
-                            'range': {
-                                'processed_crash.date_processed': {
-                                    'gte': date.isoformat(),
-                                    'lt': next_day.isoformat(),
-                                }
-                            }
-                        },
-                        {
-                            'term': {
-                                'processed_crash.product': product.lower()
-                            }
-                        },
-                        {
-                            'terms': {
-                                'processed_crash.version': [
-                                    x.lower() for x in versions
-                                ]
-                            }
-                        }
-                    ]
-                }
-            }
+        # Create a fake search object that we will use to generate the search
+        # body.
+        search = Search()
+
+        # Add filters to the search object. Note that we use that `kwargs`
+        # syntax because fields have a dot in their names.
+        filter_args = {'processed_crash.product': product.lower()}
+        search = search.filter('term', **filter_args)
+
+        filter_args = {
+            'processed_crash.version': [x.lower() for x in versions],
         }
+        search = search.filter('terms', **filter_args)
+
+        filter_args = {
+            'processed_crash.date_processed': {
+                'gte': date.isoformat(),
+                'lt': next_day.isoformat(),
+            },
+        }
+        search = search.filter('range', **filter_args)
+
+        # Turn that search body into a dict that we can send to Elasticsearch.
+        query = search.to_dict()
```

Comment:
Yes, by default the library applies an ``AND`` boolean operator between filters. So all of the filters above must be true in the query.